hyperon_das_atomdb/adapters/ram_only.py

```diff
@@ -400,8 +410,10 @@ def get_atom(
         if document is None:
             document = self._get_link(handle)
         if document:
-            atom = self._convert_atom_format(document, **kwargs)
-            return atom
+            if not kwargs.get('no_convert', False):
```

Comment:
I'm being picky with KW param names because the number of such parameters tends to grow with time. If we name them badly now, we'll end up with a big mess of parameters even we don't know the meaning of each of them. Worst... We risk to reuse some parameter name without noticing. BTW, we should start documenting all the parametrs we currently have and their meaning.